execution-service/src/main/java/org/ihtsdo/rvf/execution/service/MysqlValidationService.java

```diff
@@ -165,6 +171,13 @@ private void runAssertionTests( ValidationStatusReport statusReport, MysqlExecut
 		for (final AssertionGroup group : groups) {
 			assertions.addAll(group.getAssertions());
 		}
+
+		// filter the delta-file-required assertions out if the release does not contain the delta files
+		if (validationConfig.isMissingRf2DeltaFromRelease()) {
```

Comment:
@QuyenLy87 We could change the parameter to skipValidationForDeltaFiles

Reply:
Done